### PR TITLE
Additions to Libraries and Software

### PR DESCRIPTION
**Pull request by pjattke**
Hi @jonaschn,
I would be happy if these changes could be added to your link list.

Best regards,
Patrick

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,31 +14,39 @@
...
 
+Libraries that can be used to implement applications using (Fully) Homomorphic Encryption.
+
 - [cuHE](https://github.com/vernamlab/cuHE) - GPU-accelerated HE library for NVIDIA CUDA-Enabled GPUs.
 - [cuFHE](https://github.com/vernamlab/cuFHE) - CUDA-accelerated Fully Homomorphic Encryption Library.
 - [cuYASHE](https://github.com/cuyashe-library/cuyashe) - Based on leveled fully HE scheme YASHE for GPGPUs.
-- [FHEW](https://github.com/lducas/FHEW) - Fully HE library.
-- [HEAAN](https://github.com/snucrypto/HEAAN) -  Scheme with native support for fixed point approximate arithmetic.
+- [FHEW](https://github.com/lducas/FHEW) - A Fully HE library based on [_FHEW: Bootstrapping Homomorphic Encryption in less than a second_](https://eprint.iacr.org/2014/816).
+- [FV-NFLlib](https://github.com/CryptoExperts/FV-NFLlib) - A header-only library implementing the FV scheme.
```
Comment (jonaschn) [inline README.md]: replace FV with Fan-Vercauteren

File: README.md [hunk 2]
```diff
@@ -14,31 +14,39 @@
...
+- [HEAAN](https://github.com/kimandrik/HEAAN) -  Scheme with native support for fixed point approximate arithmetic.
```
Comment (jonaschn) [inline README.md]: https://github.com/snucrypto/HEAAN is the maintained version and therefore preferable

File: README.md [hunk 3]
```diff
@@ -14,31 +14,39 @@
...
 - [HElib](https://github.com/HomEnc/HElib) - Brakerski-Gentry-Vaikuntanathan (BGV) scheme.
+- [jLBC](http://gas.dia.unisa.it/projects/jlbc/index.html) - A Java library implementing the BGV and BV scheme.
```
Comment (jonaschn) [inline README.md]: Add direct link to https://github.com/adecaro/jlbc
This project isn't maintained anymore. Was it any helpful for you?

File: README.md [hunk 4]
```diff
@@ -14,31 +14,39 @@
...
 - [lattigo](https://github.com/ldsec/lattigo) - Go library for lattice-based crypto that implements various schemes.
+- [libScarab](https://github.com/hcrypt-project/libScarab) - C library implementing a FHE scheme using large integers.
 - [libshe](https://github.com/bogdan-kulynych/libshe) - Symmetric somewhat HE library based on DGHV scheme.
-- [NFLlib](https://github.com/quarkslab/NFLlib) - NTT-based Fast Lattice library.
+- [NFLlib](https://github.com/quarkslab/NFLlib) - NTT-based Fast Lattice library specialized on power-of-two polynomials.
 - [NuFHE](https://github.com/nucypher/nufhe) - GPU-accelerated HE library, faster than cuFHE, that implements the [tfhe](#tfhe) algorithms.
 - [PALISADE](https://git.njit.edu/palisade/PALISADE) - lattice encryption library.
 - [petlib](https://github.com/gdanezis/petlib) - Python library that implements a number of Privacy Enhancing Technologies.
-- [Pyfhel](https://github.com/ibarrond/Pyfhel) - PYthon For HELib.
+- [Pyfhel](https://github.com/ibarrond/Pyfhel) - A Python wrapper for SEAL, HElib, and PALISADE.
```
Comment (jonaschn) [inline README.md]: Add links to SEAL, HElib and PALISADE

File: README.md [hunk 5]
```diff
@@ -14,31 +14,39 @@
...
 - <a name="tfhe">[tfhe](https://github.com/tfhe/tfhe) - Faster fully HE: Bootstrapping in less than 0.1 seconds.</a>
 
 
 ## Software
 
+- [ALCHEMY](https://github.com/cpeikert/ALCHEMY) - Haskell-based DSLs and interpreters/compilers, build on top of the lattice crypto library Lol.
 - [Cingulata](https://github.com/CEA-LIST/Cingulata) - Compiler toolchain and RTE for running C++ programs over encrypted data.
 - [crypto-geofence](https://github.com/Georeactor/crypto-geofence) - Geo-fencing demo application based on Paillier scheme.
 - [nGraph-HE](https://github.com/NervanaSystems/he-transformer) - Deep Learning (DL) with HE through Intel’s DL graph compiler nGraph based on [SEAL](#SEAL).
+- [Marble](https://github.com/MarbleHE/Marble) - A C++ framework that translates nearly plaintext-code into FHE crypto libraries.
```
Comment (jonaschn) [inline README.md]: A C++ framework that translates between nearly plaintext-style user programs and FHE computations.

File: README.md (hunk 3 above)
Comment (pjattke) [reply README.md]: No, I just stumbled on it. Should it be removed as it is not maintained anymore?

Comment (jonaschn) [reply README.md]: Ok, then just remove it.

**Pull request merged**